src/AppBundle/Subscriber/NewPRInstructionCommentSubscriber.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace AppBundle\Subscriber;
+
+use AppBundle\Event\GitHubEvent;
+use AppBundle\GitHubEvents;
+use AppBundle\Issues\CommentsApiInterface;
+use AppBundle\Issues\GitHub\GitHubStatusApi;
+use AppBundle\Issues\Status;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+class NewPRInstructionCommentSubscriber implements EventSubscriberInterface
+{
+    private $commentsApi;
+
+    public function __construct(CommentsApiInterface $commentsApi)
+    {
+        $this->commentsApi = $commentsApi;
+    }
+
+    /**
+     * Adds a "Needs Review" label to new PRs.
+     *
+     * @param GitHubEvent $event
+     */
+    public function onPullRequest(GitHubEvent $event)
+    {
+        $data = $event->getData();
+        $repository = $event->getRepository();
+        if ('opened' !== $action = $data['action']) {
+            $event->setResponseData(array('unsupported_action' => $action));
+
+            return;
+        }
+
+        $needsReviewLabel = GitHubStatusApi::getNeedsReviewLabel();
+
+        $commentTexts = StatusChangeByCommentSubscriber::getValidCommentsForDisplay();
+        $commentBody = sprintf(<<<EOF
```

Comment:
Hi  @weaverryan, what do you think about use twig and configurable templates instead ?